ci/scripts/check-module-ids.sh

```diff
@@ -0,0 +1,33 @@
+#!/bin/bash
+# Copyright lowRISC contributors.
+# Licensed under the Apache License, Version 2.0, see LICENSE for details.
+# SPDX-License-Identifier: Apache-2.0
+
+# Checks that all module IDs are unique.
+
+set -e
+
+# List of all targets that need to be checked: we only consider targets that
+# produce a binary file and that depend on the status library. Since every test
+# is usually built for real targets and dv, we only check for one real target to
+# save time.
+query_elfs='
+    filter(
+        ".*prog_fpga_cw310.*",
+        kind(
+            cc_binary,
+            rdeps(
+                //sw/device/...,
+                //sw/device/lib/base:status
+            )
+        )
+    )'
+target_file=$(mktemp)
```

Comment:
I think this will create files in `/tmp` by default, won't it? Do you need to be a bit more specific about where we're making the temporary files? (And probably use a `trap` to delete them once `bazelisk.sh` is done)

Reply:
This was part of a commit from a PR that was already merged (#19062), I can address this in a follow up since @pamaury is on vacation I believe.